gp-nested-forms/gpnf-populate-parent-entry-id.php

```diff
@@ -21,9 +21,10 @@
 		} else {
 			// The first time the parent form loads there will be no cookie. Create a hash and set it as the default
 			// hash for this cookie created for this form.
-			$value = $session->make_hashcode();
+			$value = $session->get_runtime_hashcode();
```

Comment:
I noticed that when we [set the default hashcode in GPNF](https://github.com/gravitywiz/gp-nested-forms/blob/53c13ffb0e10d66874d739e46f4b0a83d774a68a/includes/class-gpnf-session.php#L280), we call `get_runtime_hashcode()`. Does it make sense to do the same here? Or does it matter?

Reply:
Yes! We added get_runtime_hashcode() later and it is the correct way to get the hashcode. 🙏

Reply:
Thank you for confirming!